src/dotnet/ExecutablePackageObtainer/ICanRestoreProject.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ExecutablePackageObtainer
+{
+    public interface ICanRestoreProject
+    {
+        void Restore(
+            FilePath tempProjectPath,
```

Comment:
Should this just say projectPath like in your other APIs?

Reply:
Fixed